### PR TITLE
Drive passenger list

### PR DESCRIPTION
**Pull request by shimanovAV**

### CODE REVIEW

File: BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java [hunk 1]
```diff
@@ -15,29 +18,63 @@
...
         return driveRepository.findAll();
     }
 
+
     @GetMapping("/{id}")
     public Drive findById(final @PathVariable Long id) {
         return driveRepository.findById(id).get();
     }
 
+    @GetMapping("/driverId/{driverId}")
```
Comment (tkuprevich) [inline BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java]: this code belonds to service layer

File: BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java [hunk 2]
```diff
@@ -72,4 +109,6 @@ public boolean delete(final @PathVariable Long id) {
         return true;
```
Comment (tkuprevich) [inline BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java]: I know this has not been changed but it always returns true even if delete not successful, void is fine here

File: BackEnd/src/main/java/com/exadel/carpoolfree/repository/DriveRepository.java [hunk 3]
```diff
@@ -2,11 +2,24 @@
...
 import com.exadel.carpoolfree.model.Drive;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 import java.time.LocalDateTime;
 import java.util.List;
 
 public interface DriveRepository extends JpaRepository<Drive, Long> {
+
+    /*@Query(value = "SELECT d " +
```
Comment (tkuprevich) [inline BackEnd/src/main/java/com/exadel/carpoolfree/repository/DriveRepository.java]: do we need this comment here?

File: BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java (hunk 1 above)
Comment (shimanovAV) [reply BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java]: Where should service be? Should I create separate package with DriveService class in it?

Comment (tkuprevich) [reply BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java]: > Where should service be? Should I create separate package with DriveService class in it?

`service` package should be a package on the same level as `controler` or 'repository', just move the code there please

**Pull request merged**